js/components/AppListComponent.jsx

```diff
@@ -68,9 +68,8 @@ var AppListComponent = React.createClass({
 
     var tableClassSet = React.addons.classSet({
       "table table-fixed": true,
-      "table-hover table-selectable":
-        this.props.collection.length !== 0 &&
-        this.props.fetchState !== States.STATE_LOADING
+      "table-hover table-selectable": this.props.collection.length !== 0 &&
+      this.props.fetchState !== States.STATE_LOADING
```

Comment:
This belongs to the line on top, as this is the continuation of the boolean expression, so the indentation should be there.